src/routeHandlers/admin.js

```diff
@@ -105,7 +106,7 @@ async function handleAdmin(
     }
 
     try {
-        req.id = req.id || req.headers['X-REQUEST-ID'] || generateUUID();
+        req.id = req.id || req.headers[REQUEST_ID_HEADER] || generateUUID();
```

Comment:
You can fix this by:
`req.headers[`${REQUEST_ID_HEADER}`]`

Reply:
Fixed